snapraid-runner.py

```diff
@@ -150,6 +155,7 @@ def load_config(args):
 
     config["smtp"]["ssl"] = (config["smtp"]["ssl"].lower() == "true")
     config["scrub"]["enabled"] = (config["scrub"]["enabled"].lower() == "true")
+    config["smart"]["enabled"] = (config["smart"]["enabled"].lower() == "true")
```

Comment:
This will raise an exception when people upgrade and don't change their config file. You need to `.get` the parts that are not clearly there.

Reply:
I'll test & confirm what happens if the config omits this section and design a method to handle if it raises an exception.

Reply:
Confirmed it does not throw an error with latest code on Windows & Linux.

Reply:
thx 👍